RubberduckTests/Grammar/VBAParserTests.cs

```diff
@@ -2136,6 +2135,26 @@ End Sub
             AssertTree(parseResult.Item1, parseResult.Item2, "//attributeStmt", matches => matches.Count == 1);
         }
 
+        [Category("Parser")]
+        [Test]
+        public void SubtractionExpressionsAreNoLetterRanges()
+        {
+            const string code = @"
+Public Sub Foo()
+    Dim a As Long
+    Dim b As Long
+    Dim z As Long
+    a = 1
+    b = 2
+    z = a-b
+    b = a-z
```

Comment:
Note that VBA normally will pretty print such expression to `b = a - z`. Though the lexer rules does not permit whitespace, I think it'll be good to test the version with whitespace to ensure we don't ever match the pretty printed expression.